test/e2e/apps/statefulset.go

```diff
@@ -2088,37 +2079,6 @@ func (m *mysqlGaleraTester) read(statefulPodIndex int, key string) string {
 	return lastLine(m.mysqlExec(fmt.Sprintf("use statefulset; select v from foo where k=\"%v\";", key), m.ss.Namespace, name))
 }
 
-type redisTester struct {
```

Comment:
`redisTester` should be safe to be removed I think. If not, maybe we should eliminate the test cases strongly required it?